### DISCUSSION TITLE
Decrypting Error Messages

### DISCUSSION THREAD

**Discussion by danicrosby** [nss-evening-cohort-14/evening-client-side#49] [Discuss a Topic]
Sometimes error messages are literally what they say, you just have to read them real slow-like, but some are a little more ambiguous. This is just a general question on if we will have an in class lesson/discussion on a best practice to decipher these? Or is this just a 'google each error' and see where you land sort of scenario? I get nervous when my terminal is all red and angry! haha

**Comment by drteresavasquez** [comment]
LOL. Yes, we will push you to research your own errors. This is how we work in the field, so knowing how to do it fast like lightening is a necessity. Now, if you search and cannot find a resolution, of course we will help you if we know the answer off hand, which if you read the errors, it will usually tell you what the problem is and in some cases what to try/run, otherwise, we will google right along with you.
